README.md

```diff
@@ -130,18 +132,16 @@ In addition to these files (and optionally their checksums), the POSEIDON.yml fi
 The `.janno` file is a tab-separated text file with a header line. It holds context information (variables/columns) for each sample (objects/rows) in a package.
 
 - A set of strictly defined core variables (defined by column name) and their possible content are documented here: [janno_columns.tsv](https://github.com/poseidon-framework/poseidon-schema/blob/master/janno_columns.tsv)
-- A `.janno` file CAN have all of these core variables, or only a subset of them.
+- A `.janno` file MAY have all of these core variables, or only a subset of them.
 - Only three columns MUST be present to make the file valid: **Poseidon_ID**, **Group_Name** and **Genetic_Sex**
-- Arbitrary columns not defined here CAN be added as long as their column names do not clash with the defined ones.
+- Arbitrary columns not defined here MAY be added as long as their column names do not clash with the defined ones.
 - The column order is irrelevant.
-- If information is unknown or a variable does not apply for a certain sample, then the respective cell(s) can be filled with the NULL value `n/a` or simply an empty string.
+- If information is unknown or a variable does not apply for a certain sample, then the respective cell(s) MAY be filled with the NULL value `n/a` or simply an empty string.
```

Comment:
I would remove `NULL value` here and simply write "filled with `n/a` or simply an empty string. The term `NULL value` sounds like database speak and might confuse people. In worst cases people might literally want to fill "NULL" or so.

Reply:
Good point. I changed it as you suggested.